apps/vpnmanager/next.config.mjs

```diff
@@ -0,0 +1,38 @@
+import { withSentryConfig } from "@sentry/nextjs";
```

Comment:
Does this **need** to be `.mjs`?

Reply:
@kilemensi it doesn't have to be, I thought  using `import/export` is cooler than `require/module.exports`